model/pytorch/dffml_model_pytorch/pytorch_base.py

```diff
@@ -252,9 +266,9 @@ async def train(self, sources: Sources):
 
```

Comment:
@pdxjohnny I think it'd be better if we change these epoch debug statements to print or logger.info? Also, we might need to look at the logger.info statements in `dffml/df` to not output big arrays as it currently does.